mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -156,6 +156,124 @@ func TestAccContainerNodePool_noName(t *testing.T) {
 	})
 }
 
+func TestAccContainerNodePool_withNoSpecifiedLoggingVariant(t *testing.T) {
+	t.Parallel()
+
+	cluster := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+	nodePool := fmt.Sprintf("tf-test-nodepool-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerNodePoolDestroyProducer(t),
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccContainerNodePool_basic(cluster, nodePool),
+				Check: resource.TestCheckResourceAttr("google_container_node_pool.np",
+					"node_config.0.logging_variant",
+					"DEFAULT"),
+			},
+			resource.TestStep{
+				ResourceName:      "google_container_node_pool.np",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccContainerNodePool_withDefaultLoggingVariant(t *testing.T) {
```

Comment:
Container cluster tests are pretty expensive in terms of time it takes them to run. Although we want to exercise every field, we don't need to test every value for every field. I'd recommend removing this test since it uses the default value, so we wouldn't be able to tell if the API was honoring the value we're setting.

Reply:
Ack.